DESCRIPTION

```diff
@@ -1,100 +1,100 @@
-VignetteBuilder: knitr
-Collate:
-    'aaa-enums.R'
-    'aaa-utilities.R'
-    'aggregation-input.R'
-    'aggregation-summary.R'
-    'atom-plots.R'
-    'boxwhisker-datamapping.R'
-    'boxwhisker-get-measure.R'
-    'boxwhisker-plotconfiguration.R'
-    'datamapping-grouping.R'
-    'datamapping-groupmapping.R'
-    'datamapping-range.R'
-    'datamapping-xy.R'
-    'datamapping-xygroup.R'
-    'ddiratio-datamapping.R'
-    'error-checks.R'
-    'font.R'
-    'histogram-datamapping.R'
-    'label.R'
-    'messages.R'
-    'obs-vs-pred-datamapping.R'
-    'observed-data-mapping.R'
-    'pkratio-datamapping.R'
-    'pkratio-get-measure.R'
-    'plot-boxwhisker.R'
-    'plot-ddiratio.R'
-    'plot-grid.R'
-    'plot-histogram.R'
-    'plot-obs-vs-pred.R'
-    'plot-pkratio.R'
-    'plot-res-vs-pred.R'
-    'plot-timeprofile.R'
-    'plot-tornado.R'
-    'plotconfiguration-axis.R'
-    'plotconfiguration-background.R'
-    'plotconfiguration-export.R'
-    'plotconfiguration-label.R'
-    'plotconfiguration-legend.R'
-    'plotconfiguration.R'
-    'themes.R'
-    'timeprofile-datamapping.R'
-    'timeprofile-helper.R'
-    'tlf-env.R'
-    'tornado-datamapping.R'
-    'tornado-plotconfiguration.R'
-    'utilities-aesthetics.R'
-    'utilities-axis.R'
-    'utilities-background.R'
-    'utilities-export.R'
-    'utilities-label.R'
-    'utilities-legend.R'
-    'utilities-mapping.R'
-    'utilities-theme.R'
-    'utils.R'
+Package: tlf
+Type: Package
+Title: TLF Library
+Version: 1.4.0
+Authors@R: 
+    c(person(given = "Open-Systems-Pharmacology Community",
+             role = "cph"),
+      person(given = "Michael",
+             family = "Sevestre",
+             role = c("aut", "cre"),
+             email = "michael@design2code.ca"),
+      person(given = "Pierre",
+             family = "Chelle",
+             role = "aut",
+             email = "chelle.pierre@gmail.com"),
+      person(given = "Abdullah",
+             family = "Hamadeh",
+             role = "aut",
+             email = "aoh123@gmail.com"),
+      person(given = "Indrajeet",
+             family = "Patil",
+             role = "ctb",
+             email = "patilindrajeet.science@gmail.com",
+             comment = c(ORCID = "0000-0003-1995-6531", Twitter = "@patilindrajeets")))
+Description: An implementation of the Table, Listing and Figure concepts in R. This library provides
+    a standardized approach to creating tables and graphs based on conventions over configurations.
+License: GPL-2 | file LICENSE
+URL: https://github.com/open-systems-pharmacology/tlf-library
+BugReports: https://github.com/open-systems-pharmacology/tlf-library/issues
+Imports: 
+    ggplot2 (>= 3.3.0),
+    R6,
+    jsonlite,
+    ospsuite.utils,
+    patchwork
+Depends: 
+    R (>= 3.6)
+Encoding: UTF-8
+RoxygenNote: 7.2.0
+Roxygen: list(markdown = TRUE)
+Suggests:
+    knitr,
+    rmarkdown,
+    testthat (>= 3.0.3),
+    vdiffr (>= 1.0.0)
+VignetteBuilder: knitr
+Collate: 
```

Comment:
Are you using roxygen to create the collate field, or do it manually?

Reply:
Roxygen

That's why I have added `#' @include` directive which was missing before. That's what created problems.